script/tx_validation.md

```diff
@@ -0,0 +1,14 @@
+# Transaction Validation
```

Comment:
I'm not as sure about this page as I am the others. I wonder if adding some of this to the "Script Success and Failures" would make sense? There is also the issue of the new nomenclature for locking/unlocking scripts as mentioned by @vostrnad.

Overall I feel like this page might be better left for another PR where we can iterate on it there. Do you mind removing it from this one and we can maybe iterate over the idea in a different PR?

Reply:
Sure. I agree it needs some work, including the updating the terminology.